yarn.lock

```diff
@@ -17610,14 +17610,30 @@ react-native-safe-area-context@3.3.2:
   resolved "https://registry.yarnpkg.com/react-native-safe-area-context/-/react-native-safe-area-context-3.3.2.tgz#9549a2ce580f2374edb05e49d661258d1b8bcaed"
   integrity sha512-yOwiiPJ1rk+/nfK13eafbpW6sKW0jOnsRem2C1LPJjM3tfTof6hlvV5eWHATye3XOpu2cJ7N+HdkUvUDGwFD2Q==
 
-react-native-screens@^3.3.0, react-native-screens@^3.4.0, react-native-screens@~3.10.1:
+react-native-screens@^3.11.1:
+  version "3.13.1"
+  resolved "https://registry.yarnpkg.com/react-native-screens/-/react-native-screens-3.13.1.tgz#b3b1c5788dca25a71668909f66d87fb35c5c5241"
+  integrity sha512-xcrnuUs0qUrGpc2gOTDY4VgHHADQwp80mwR1prU/Q0JqbZN5W3koLhuOsT6FkSRKjR5t40l+4LcjhHdpqRB2HA==
+  dependencies:
+    react-freeze "^1.0.0"
+    warn-once "^0.1.0"
+
+react-native-screens@^3.4.0:
```

Comment:
Can't 3.4.0 and 3.11.1 be casted to a single 3.13.1?

Just curious why before we had three different libraries pointing to single library version, and now it seems we have 3 different versions of react-native-screens 🤔

Reply:
These are some typical monorepo issues. But we should not 3.13.1, since 3.11.1 is the version shipped with the latest Expo SDK 45. I am not sure where RNS 3.4.0 is coming from, usually this issues disappear when you delete the yarn.lock and reinstall it

Reply:
@WoLewicki 

```
➜  react-navigation git:(main) ✗ yarn why react-native-screens
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "react-native-screens"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "react-native-screens@3.11.1"
info Has been hoisted to "react-native-screens"
info Reasons this module exists
   - "workspace-aggregator-37e960f9-a2b1-4db0-b892-7f2b0e76bc0e" depends on it
   - Hoisted from "_project_#@react-navigation#bottom-tabs#react-native-screens"
   - Hoisted from "_project_#@react-navigation#drawer#react-native-screens"
   - Hoisted from "_project_#@react-navigation#stack#react-native-screens"
   - Hoisted from "_project_#@react-navigation#example#react-native-screens"
info Disk size without dependencies: "1.84MB"
info Disk size with unique dependencies: "1.92MB"
info Disk size with transitive dependencies: "1.92MB"
info Number of shared dependencies: 2
=> Found "@react-navigation/native-stack#react-native-screens@3.13.1"
info This module exists because "_project_#@react-navigation#native-stack" depends on it.
info Disk size without dependencies: "2.25MB"
info Disk size with unique dependencies: "2.33MB"
info Disk size with transitive dependencies: "2.33MB"
info Number of shared dependencies: 2
=> Found "react-native-reanimated#react-native-screens@3.10.2"
info This module exists because "_project_#@react-navigation#drawer#react-native-reanimated" depends on it.
info Disk size without dependencies: "1.78MB"
info Disk size with unique dependencies: "1.86MB"
info Disk size with transitive dependencies: "1.86MB"
info Number of shared dependencies: 2
✨  Done in 1.19s.
```

I'll try to regenerate the lock file

Reply:
But another issue is that react-native-reanimated adds a peer to 3.10.2. 
Should we add a yarn resolution to package.json ?

Reply:
@tomekzaw @piaskowyk is there a reason for this peer dep in `reanimated`?

Reply:
I fixed all the package.json @WoLewicki 
The last piece is reanimated.

```
➜  react-navigation git:(main) yarn why react-native-screens
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "react-native-screens"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "react-native-screens@3.11.1"
info Has been hoisted to "react-native-screens"
info Reasons this module exists
   - "workspace-aggregator-7ec20abb-c574-406e-97c1-14d823890d28" depends on it
   - Hoisted from "_project_#@react-navigation#bottom-tabs#react-native-screens"
   - Hoisted from "_project_#@react-navigation#drawer#react-native-screens"
   - Hoisted from "_project_#@react-navigation#native-stack#react-native-screens"
   - Hoisted from "_project_#@react-navigation#stack#react-native-screens"
   - Hoisted from "_project_#@react-navigation#example#react-native-screens"
info Disk size without dependencies: "1.84MB"
info Disk size with unique dependencies: "1.92MB"
info Disk size with transitive dependencies: "1.92MB"
info Number of shared dependencies: 2
=> Found "react-native-reanimated#react-native-screens@3.10.2"
info This module exists because "_project_#@react-navigation#drawer#react-native-reanimated" depends on it.
info Disk size without dependencies: "1.78MB"
info Disk size with unique dependencies: "1.86MB"
info Disk size with transitive dependencies: "1.86MB"
info Number of shared dependencies: 2
✨  Done in 1.24s.
```

Reply:
@hirbod looks better 👍 But why ^3.4.0 is pointed to 3.10.2 now, whereas we have 3.11.1 in the project? I thought ^3.4.0 means 3.x library (according to npm sematic versioning), so it should re-use 3.11.1 version? Or I'm missing something?

Reply:
Not sure. 3.4.0 is not being hoisted regarding to yarn why. It's 3.11.1 and 3.10.2 because of rea

Reply:
@WoLewicki looks like the peer has been removed in 2.8.0
https://github.com/software-mansion/react-native-reanimated/commit/5601bc62cd5e1e2b4a1c41ce3e9d7059207a37d9

Shall I bump REA to 2.8.0? It is also the shipped version with SDK 45

Reply:
Yeah, I think it is a good idea to bump it too if nothing breaks then

Reply:
@WoLewicki upgrading REA to 2.8.0 fixes the issues but the example app will stuck on SplashScreen. Not sure if thats really the scope of this PR for me to upgrade to Expo SDK 45 now? I could do this in another PR

Reply:
That's weird. Does the app crash then?

Reply:
It does look like that. It just stuck on the SplashScreen as soon as I bump REA to 2.8.0. Guess some work has to be done (upgrading to Expo 44). I think there had been changes with auto installing which don't work with SDK 44